script/bulk_import/generic_bulk.rb

```diff
@@ -100,6 +100,7 @@ def execute
     import_badge_groupings
     import_badges
     import_user_badges
+    import_anniversary_user_badges
```

Comment:
I went with making this specific to anniversary badges for now, rather than trying to generalize it to other badges. I’m still reviewing the other badge types to see if it might be necessary to add support for them as well.

Reply:
Makes sense. I believe this is the only badge in core that needs this kind of special handling.